app/routes/errors.py

```diff
@@ -191,8 +195,10 @@ def too_many_feedback_requests(
 
 
 @errors_blueprint.app_errorhandler(SupplementaryDataRequestFailed)
+@errors_blueprint.app_errorhandler(MissingSupplementaryDataKey)
+@errors_blueprint.app_errorhandler(InvalidSupplementaryData)
 def supplementary_data_request_failed(
-    exception: SupplementaryDataRequestFailed,
+    exception: SupplementaryDataRequestFailed | MissingSupplementaryDataKey,
```

Comment:
Should also add `InvalidSupplementaryData`

Reply:
Added 👍